plugins/source/firestore/resources/plugin/plugin.go

```diff
@@ -5,12 +5,19 @@ import (
 	"github.com/cloudquery/plugin-sdk/v4/plugin"
 )
 
-var version = "development"
```

Comment:
The fact that `version` was unexported here was actually a bug, as even official versions of the plugin right now report their version to be `development`:

```
./plugin --version
firestore version development
```